<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShadowrunTool</class>
 <widget class="QMainWindow" name="ShadowrunTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShadowrunTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>641</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>[Allgemein]</string>
     </attribute>
     <widget class="QTableWidget" name="uiSkillTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>320</width>
        <height>630</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>[Angriff]</string>
     </attribute>
     <widget class="QTableWidget" name="uiSkillTable_2">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>2</y>
        <width>320</width>
        <height>630</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>[Daten Verarbeitung]</string>
     </attribute>
     <widget class="QTableWidget" name="uiSkillTable_3">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>2</y>
        <width>320</width>
        <height>630</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>[Schleicher]</string>
     </attribute>
     <widget class="QTableWidget" name="uiSkillTable_4">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>2</y>
        <width>320</width>
        <height>630</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>[Firewall]</string>
     </attribute>
     <widget class="QTableWidget" name="uiSkillTable_5">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>2</y>
        <width>320</width>
        <height>630</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="skilldescription_lab">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>561</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Skill Beschreibung</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Angriff</string>
    </property>
    <widget class="QSpinBox" name="spinAttack">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="value">
      <number>7</number>
     </property>
    </widget>
    <widget class="QComboBox" name="prog_one">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>75</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="progDescribBox_one">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>75</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Schleicher</string>
    </property>
    <widget class="QComboBox" name="prog_two">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>75</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="progDescribBox_two">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>75</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinSleep">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="value">
      <number>7</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>260</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Datenverarbeitung</string>
    </property>
    <widget class="QComboBox" name="prog_three">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>75</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QToolButton" name="progDescribBox_three">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>75</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>260</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Firewall</string>
    </property>
    <widget class="QComboBox" name="prog_four">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>75</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="progDescribBox_four">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>75</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinFirewall">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="currInventory">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>10</y>
      <width>241</width>
      <height>241</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Inventar</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1088</x>
      <y>269</y>
      <width>99</width>
      <height>103</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="conf_std">
       <property name="text">
        <string>Standard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="conf_schl">
       <property name="text">
        <string>Schleicher</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="conf_dv">
       <property name="text">
        <string>Datenverarb</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="conf_var">
       <property name="text">
        <string>Variable</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>380</y>
      <width>341</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Initiative &amp; Modifikationen</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>321</width>
       <height>248</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>INI-Wert:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="iniValue">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="ini_roll">
        <property name="text">
         <string>INI-Roll</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="coldsim">
        <property name="text">
         <string>Kaltes Sim</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="additionalIniDie">
        <property name="text">
         <string>+1w6</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="hotsim">
        <property name="text">
         <string>Heißes Sim</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="arsim">
        <property name="text">
         <string>AR</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="endRound">
        <property name="text">
         <string>Runde fertig</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Hardening</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Induk. Receiv</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Multi. Co-Proc.</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="status_sim">
        <property name="text">
         <string>Status: 
</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>380</y>
      <width>301</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Würfel-Tool</string>
    </property>
    <widget class="QPushButton" name="subfivedice">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>93</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>-5 Dice</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addtendice">
     <property name="geometry">
      <rect>
       <x>196</x>
       <y>87</y>
       <width>93</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>+10 Dice</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addtwodice_2">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>87</y>
       <width>93</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>+2 Dice</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="dicecount">
     <property name="geometry">
      <rect>
       <x>138</x>
       <y>20</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QPushButton" name="addfivedice">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>87</y>
       <width>93</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>+5 Dice</string>
     </property>
    </widget>
    <widget class="QPushButton" name="subtendice">
     <property name="geometry">
      <rect>
       <x>196</x>
       <y>140</y>
       <width>93</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>-10 Dice</string>
     </property>
    </widget>
    <widget class="QPushButton" name="diceroll">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>20</y>
       <width>123</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Roll the Dice</string>
     </property>
    </widget>
    <widget class="QPushButton" name="subtwodice_2">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>139</y>
       <width>93</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>-2 Dice</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>381</y>
      <width>201</width>
      <height>205</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="rolledDice">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string> 1: 
 2: 
 3: 
 4: 
 5: 
 6: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="diceresults">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>Würfel Egebnisse: 
</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="IniBar">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>600</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>21</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="format">
     <string>Restliche Ini: %p</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionInventar"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInventar">
   <property name="text">
    <string>Inventar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
